cmd/lookout/event.go

```diff
@@ -64,13 +64,13 @@ func (c *EventCommand) resolveRefs() (*lookout.ReferencePointer, *lookout.Refere
 
 	fromRef := lookout.ReferencePointer{
 		InternalRepositoryURL: "file://" + fullGitPath,
-		ReferenceName:         plumbing.ReferenceName(c.RevFrom),
+		ReferenceName:         plumbing.HEAD,
```

Comment:
Why is this change needed?

Reply:
Because it's a bug otherwise - the purpose of this field to construct a [RefSpec](https://git-scm.com/book/en/v2/Git-Internals-The-Refspec), so it has to be reference, and not a commit hash as before.

The reason it did not fail at runtime when using SDK binary is that we never fetched. Lookout SDK, as opposed to serve, did not use LibraryCommitLoader, which before this change, was the only one way to validate it through actually trying to fetch a repo.